plugins/varnish/varnish4_multigraph

```diff
@@ -0,0 +1,453 @@
+#!/usr/bin/python3 -tt
```

Comment:
What is the effect of `-tt`? I could not find it in the man page (but `python3` accepts it - thus it seems to work).

btw.: I remember vaguely, that shebang parameters are not evaluated on every platform. Thus I would prefer to remove it. But maybe I am mistaken ...